include/gauge_tools.h

```diff
@@ -145,6 +145,21 @@ namespace quda
   */
   void WFlowStep(GaugeField &out, GaugeField &temp, GaugeField &in, double epsilon, QudaGaugeSmearType smear_type);
 
+  /**
+     @brief Apply intermediary Wilson Flow steps W1, W2 or Vt to the gauge field.
+     This routine assumes that the input and output fields are
+     extended, with the input field being exchanged prior to calling
+     this function.  On exit from this routine, the output field will
+     have been exchanged.
+     @param[out] dataDs Output smeared field
```

Comment:
I see there are typos in the doxygen here, inherited from the typos in `WFlowStep` above.  Can you this these?  E.g., 
```
     @param[out] dataDs Output smeared field
```
should be
```
     @param[out] out Output smeared field
```